src/App.js

```diff
@@ -1,6 +1,21 @@
-// 해당 위치에 필요한 컴포넌트를 바탕으로 직접 google 검색 페이지를 개발하시면 됩니다!
+import Album from "./component/Album";
+import GlobalStyle from "./component/GlobalStyle";
+import Header from "./component/Header";
+import MiddleText from "./component/MiddleText";
+import Footer from "./component/Footer";
+// 해당 위치에 필요한 컴포넌트를 바탕으로 직접 Bootstrap Album 페이지를 개발하시면 됩니다!
 function App() {
-  return <>google</>;
+    return (
+        <>
+            <GlobalStyle />
+            <Header />
+            <MiddleText></MiddleText>
+            <Album></Album>
+            <Footer></Footer>
```

Comment:
```suggestion
            <MiddleText/>
            <Album/>
            <Footer/>
```

self-closing tag 로 축약해볼까요?

Reply:
<img width="329" alt="image" src="https://user-images.githubusercontent.com/64801796/168457543-9cc8b949-85b0-4ca8-affc-03b58d17c8d7.png">
수정 완료 했습니다!